python/lsst/pipe/analysis/coaddAnalysis.py

```diff
@@ -433,21 +433,8 @@ def runDataRef(self, patchRefList, subdir="", cosmos=None):
                 skyObjCatAll = unforced[unforced["merge_peak_sky"]].copy(deep=True)
             if self.config.doPlotSkyObjects:
                 baseGoodSky = (unforced["merge_peak_sky"] & (unforced["base_InputCount_value"] > 0)
-                               & ~unforced["base_PixelFlags_flag_edge"])
+                               & (unforced["deblend_nChild"] == 0) & ~unforced["base_PixelFlags_flag_edge"])
```

Comment:
I would not use this flag. If this was a scarlet run and a parent failed to deblend, then that parent will have `deblend_nChild==0` and `deblend_failed=True`, or if the footprint was too large `deblend_skipped=True`. Better to just use the `detect_isDeblendedSource` flag, which is designed to filter those cases out for both `meas_extensions_scarlet` and `meas_deblender`.